tst/testbugfix/2017-06-20-blist-list-len.tst

```diff
@@ -0,0 +1,17 @@
+# Actual bug -- checking lists are the same length
```

Comment:
Also, now about instead of a bugfix file, use this as start for a `blist.tst`? I.e. Just move&rename the file